website/docs/docs/dbt-versions/release-notes/04-Nov-2022/ide-features-ide-deprecation.md

```diff
@@ -0,0 +1,32 @@
+---
+
+title: "Extra features in new IDE, and classic IDE deprecation"
+id: "ide-features-ide-deprecation"
+description: "Enhancement and Deprecation: Extra features in new IDE, and classic IDE deprecation"
+sidebar_label: "Enhancement and Deprecation: Extra features in new IDE, and classic IDE deprecation"
+tags: [Nov-29-2022]
+
+---
+
+### Extra features in new and refreshed IDE
+
+The new and refreshed version of the Cloud IDE has launched four brand-new additional features, making it easier and faster for you to develop in the IDE.
+
+The new features are:
+
+- **Formatting** &mdash; Format your files with a click of a button, powered by sqlfmt.
+- **Git diff view** &mdash; Ability to see what you've changed in a file before you make a pull request.
+- **dbt autocomplete** &mdash; There are four new types of autocomplete features to help you develop faster:
+    - Use ref to autocomplete your model names
+    - Use source to autocomplete your source name + table name
+    - Use macro to autocomplete your arguments
+    - Use env var to autocomplete env var
+- **Dark mode**	&mdash;  Use dark mode in the Cloud IDE for a great viewing experience in low-light environments.
```

Comment:
```suggestion
- **Dark mode**	&mdash;  Use dark mode in the dbt Cloud IDE for low-light environments.
```

Reply:
sugg: 

- conciseness 
- when referencing a product, use the full product name "dbt Cloud" (or "dbt Core")